lib/Auth/Source/Twitter.php

```diff
@@ -47,11 +36,11 @@ class Twitter extends Auth\Source
     /** @var string */
     private string $secret;
 
-    /** @var bool */
-    private bool $force_login;
+    /** @var string */
+    private string $scope;
```

Comment:
Not used anywhere?

Reply:
Thanks, good catch! Can I get your opinion on the attributes ^^ as well?

Reply:
What exactly?

Reply:
About the auth attributes and the state, but you already answered it.. I think I should have refreshed
Thanks